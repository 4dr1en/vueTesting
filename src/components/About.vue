<script setup></script>

<template>
  <div>
    <h1>ABOUT US</h1>
  </div>
</template>

<style lang="scss"></style>
