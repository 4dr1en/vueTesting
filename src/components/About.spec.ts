import About from "./About.vue";
import { mount } from "@vue/test-utils";
import { expect, describe, it } from "vitest";

describe("About", () => {
  it("should match the snapshot", () => {
    const wrapper = mount(About);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
