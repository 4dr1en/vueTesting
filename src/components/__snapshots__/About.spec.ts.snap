// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`About > should match the snapshot 1`] = `
"<div>
  <h1>ABOUT US</h1>
</div>"
`;
